sdks/python/src/opik/integrations/langchain/openai_run_helpers.py

```diff
@@ -24,7 +24,22 @@ def get_llm_usage_info(run_dict: Optional[Dict[str, Any]] = None) -> LLMUsageInf
 
 def _try_get_token_usage(run_dict: Dict[str, Any]) -> Optional[UsageDict]:
     try:
-        token_usage = run_dict["outputs"]["llm_output"]["token_usage"]
+        if run_dict["outputs"]["llm_output"] is not None:
+            token_usage = run_dict["outputs"]["llm_output"]["token_usage"]
+
+        # streaming mode handling
+        else:
+            token_usage_full = run_dict["outputs"]["generations"][-1][-1]["message"][
```

Comment:
Let's maybe keep some explanations here? An example of data? This chain is quite hard to understand.
Btw is this code executed at the end of the streaming?